concepts/array-analysis/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for array-analysis concept",
+  "blurb": "Javascript provides a number of useful array methods for traversal, analysis and mutation.",
```

Comment:
Nice. Perhaps traversal and mutation should be left out here, given the concept name.